src/durabletask/azext_durabletask/_params.py

```diff
@@ -7,7 +7,10 @@
 
 # pylint: disable=too-many-lines
 # pylint: disable=too-many-statements
-
+# pylint: disable=line-too-long
 
 def load_arguments(self, _):  # pylint: disable=unused-argument
-    pass
+    with self.argument_context('durabletask orchestration list') as c:
+        c.argument('max_items', options_list=['--max-items', '-i'], help='The maximum number of orchestrations to return')
```

Comment:
Is this an optional argument?  (I would think it should be.)

Reply:
It is optional and defaults to 100